roles/linux-executor/defaults/main/install_python_vars.yml

```diff
@@ -2,6 +2,10 @@ python_versions:
   - 2.7.18
   - 3.10.5
 
+python2_version: 2.7.18
+
+python3_version: 3.10.5
+
 pyenv_version: "v2.3.3"
 
-pyenv_dir: '/opt/circleci/.pyenv'
+pyenv_dir: '/home/circleci/.pyenv'
```

Comment:
Why are we changing this? Just checked the current (non-Ansible) image and `/opt/circleci` seems correct:

```
circleci@default-4d65a00a-da94-4328-9fb3-835bd8721fd8:~$ which pyenv
/opt/circleci/.pyenv/bin/pyenv
```

Reply:
i realize that it's in /opt for the non-ansible image, however, i've tried several ways to get around this constraint, but no matter what i do, pyenv ultimately downloads the versions of python to /home/circleci rather than what we are specifying as $PYENV_ROOT and results in an error.

As of now, this ultimately does work, but I didn't want to spend more time than I already have to fix something minor. I can look into this separately and figure out why the set environment variable is not behaving the way it should some other time.

Additionally, I consulted some ansible modules, but even those do not provide support for installing outside of the home directory, which suggests additional hoops that we would need to jump through